pkgs/os-specific/linux/nftables/default.nix

```diff
@@ -4,7 +4,7 @@
 , gmp, jansson, readline
 , withDebugSymbols ? false
 , withPython ? false , python3
-, withXtables ? false , iptables
+, withXtables ? true , iptables
```

Comment:
I still think this should refer to `iptables-compat` instead based on intuition... was the xtables support of nftables tested? what does it even do?

Reply:
`withXtables = false` https://github.com/NixOS/nixpkgs/pull/81172#issuecomment-592500459
`withXtables = true` https://github.com/NixOS/nixpkgs/pull/81172#issuecomment-592927273
You should consult upstream for what exactly it does. This is enabled on Arch, I don't see why we shouldn't.
`iptables-nftables-compat` is an alias and probably should go away.

Reply:
http://git.netfilter.org/nftables/tree/configure.ac#n94

```
AC_ARG_WITH([xtables], [AS_HELP_STRING([--with-xtables],
            [Use libxtables for iptables interaction])],
	    [], [with_xtables=no])
AS_IF([test "x$with_xtables" != xno], [
PKG_CHECK_MODULES([XTABLES], [xtables >= 1.6.1])
AC_DEFINE([HAVE_LIBXTABLES], [1], [0])
])
AM_CONDITIONAL([BUILD_XTABLES], [test "x$with_xtables" = xyes])
```
it doesn't make sense to link that against the iptables-based-on-nft variant.

Reply:
> it doesn't make sense to link that against the iptables-based-on-nft variant.

I doubt it matters, and I don't want two `iptables` packages in system closure. You should ask upstream if it makes sense or not. I don't really care, as long as it works as expected (and it does, see https://github.com/NixOS/nixpkgs/pull/81172#issuecomment-592927273).
Do you think Arch and Debian compile nft-compatible AND legacy variants of `iptables` package just to enable this option?

Reply:
ok, I finally looked into the iptables source code, `libxtables` seems to be completely independent of the ´nftables´ support, that seems to be everything that matters here, concern resolved.